src/components/creators/MobileIncreaseSubRewards.tsx

```diff
@@ -0,0 +1,30 @@
+import { SpaceData } from '@subsocial/api/types'
+import clsx from 'clsx'
+import Link from 'next/link'
+import { ComponentProps } from 'react'
+import { useFetchStakeData } from 'src/rtk/features/creators/stakesHooks'
+import { getActiveStakingLinks } from 'src/utils/links'
+import { useMyAddress } from '../auth/MyAccountsContext'
+import styles from './MobileIncreaseSubRewards.module.sass'
+
+export type MobileIncreaseSubRewardsProps = ComponentProps<'div'> & {
+  space: SpaceData
+}
+
+export default function MobileIncreaseSubRewards(props: MobileIncreaseSubRewardsProps) {
+  const myAddress = useMyAddress()
+  const { data } = useFetchStakeData(myAddress ?? '', props.space.id)
+  if (!data?.hasStaked) return null
+
+  return (
+    <div {...props} className={clsx(props.className, styles.MobileIncreaseSubRewards)}>
+      <span className={styles.Title}>Increase SUB rewards</span>
+      <Link href={getActiveStakingLinks().learnMore} passHref>
```

Comment:
why `getActiveStakingLinks().learnMore` is function, not just array? 

It is static data